test/Performance/HelloWorld.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
```

Comment:
Would it make sense to change the filename here to represent what this test is measuring?

Hello World doesn't tell me much about what this file tests, but `dotnet-build.Performance.Tests.cs` might (as an example).